package.json

```diff
@@ -79,14 +79,15 @@
     "rollup-pluginutils": "^2.3.3",
     "rxjs": "^6.3.3",
     "saslprep": "^1.0.2",
+    "season": "^6.0.2",
```

Comment:
Is this a typescript dep?

Reply:
Ah, thanks that was left over from another branch - removed.